docs/java/java-testing.md

```diff
@@ -169,6 +169,9 @@ Currently the supported configurations are:
 - **sourcePaths**: Specify the extra source paths when debugging the tests.
 - **vmArgs**: Specify the extra options and system properties for the JVM.
 - **workingDirectory**: Specify the working directory when running the tests.
+- **testKind**: Specify the targeting test framework for this test configuration. Supported values are `junit`, `testng`.
+- **filters**: Specify the test filters.
+  - **tags**: Specify the tags to be included or excluded. Tags having `!` as the prefix will be **excluded**. Note: This setting only takes effect when `testKind` is set to `junit`
```

Comment:
Did you mean for "tags" to be indented one level?

![image](https://user-images.githubusercontent.com/12818376/207469506-c76ab636-2b75-4c17-8aaf-261317459592.png)

Reply:
Yes, it belongs to `filters`.

The setting looks like in this way

```
...
filters: {
    tags: ["foo", "!bar"]
}
...
```